extension/observer/dockerobserver/extension.go

```diff
@@ -308,9 +309,15 @@ func (d *dockerObserver) endpointForPort(portObj nat.Port, c *dtypes.ContainerJS
 		id = observer.EndpointID(fmt.Sprintf("%s:%d", c.ID, port))
 	}
 
+	imageRef, err := dcommon.ParseImageName(c.Config.Image)
+	if err != nil {
+		d.logger.Error("could not parse container image name", zap.Error(err))
+	}
+
 	details := &observer.Container{
 		Name:        strings.TrimPrefix(c.Name, "/"),
-		Image:       c.Config.Image,
+		Image:       imageRef.Repository,
+		Tag:         imageRef.Tag,
```

Comment:
Could be worth adding as a default attribute later? https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/receivercreator/factory.go#L57

Reply:
Definitely - I was going to open a subsequent PR to add that, I was trying to only touch observer pieces in this PR.